app/src/main/java/ai/elimu/filamu/ui/video/VideoActivity.kt

```diff
@@ -90,16 +90,15 @@ class VideoActivity : AppCompatActivity() {
 
                     if (playbackState == Player.STATE_ENDED) {
                         videoTitle = intent.getStringExtra(EXTRA_KEY_VIDEO_TITLE) ?: ""
-                        val extraData = JSONObject().apply {
-                            put(ANALYTICS_PLAYBACK_POSITION, videoPlayer.duration)
-                        }
                         LearningEventUtil.reportVideoLearningEvent(
                             videoGson = VideoGson().apply {
                                 id = videoId
                                 title = videoTitle
                             },
-                            additionalData = extraData,
-                            learningEventType = LearningEventType.VIDEO_COMPLETED,
+                            additionalData = JSONObject().apply {
+                                put("eventType", LearningEventType.VIDEO_COMPLETED)
+                                put("video_playback_position_ms", videoPlayer.duration)
```

Comment:
@jo-elimu What's the reason for removing the predefined constant `ANALYTICS_PLAYBACK_POSITION` and use hardcoded text value instead?

Reply:
@tuancoltech Just for making it faster/easier to read the code.